src/main/resources/application.yaml

```diff
@@ -14,3 +14,6 @@ spring:
     schemas: public
     locations: db/migration
     baselineOnMigrate: true
+  security:
+    user:
+      password: "user"
```

Comment:
Sensitive data should be stored as an env-variable and have a default value e.g. `${USER_PASSWORD:user}`

Reply:
Fixed